packages/core-components/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@okta/core.components",
+  "version": "0.0.1",
```

Comment:
We'll want to make sure than anything versioned matches our current/planned ODS release.